caiman_napari/main_offline_gui.py

```diff
@@ -62,16 +64,22 @@ def __init__(self, napari_viewer: Viewer):
         ## Change parameters displayed in param text box upon change in selection
         self.ui.listWidgetItems.currentRowChanged.connect(self.set_params_text)
         ## On double click of item in listwidget, load results and display
-        self.ui.listWidgetItems.doubleClicked.connect(self.show_cnmf_results)
+        self.ui.listWidgetItems.doubleClicked.connect(self.load_output)
 
-    @use_open_file_dialog('Choose image file', '', ['*.tiff', '*.tif', '*.btf'])
+    @use_open_file_dialog('Choose image file', '', ['*.tiff', '*.tif', '*.btf', '*.mmap'])
     def open_movie(self, path: str, *args, **kwargs):
         if not self.clear_viewer():
             return
 
         self.input_movie_path = path
         self.viewer.open(self.input_movie_path)
 
+
+        #Yr, dims, T = cm.load_memmap(path)
```

Comment:
I think it would be useful to manually open memmaps too. You could just add an `if` statement to see if the path given is a memmap, then use the stuff that is currently commented out

Reply:
Ah good point!